docs/specifications/tests/Nameserver-TP/nameserver11.md

```diff
@@ -3,91 +3,142 @@
 ## Test case identifier
 **NAMESERVER11** 
 
+## Table of contents
+
+* [Objective](#Objective)
+* [Scope](#Scope)
+* [Inputs](#Inputs)
+* [Summary](#Summary)
+* [Test procedure](#Test-procedure)
+* [Outcome(s)](#Outcomes)
+* [Special procedural requirements](#Special-procedural-requirements)
+* [Intercase dependencies](#Intercase-dependencies)
+* [Terminology](#Terminology)
+
 ## Objective
 
 EDNS is a mechanism to announce capabilities of a DNS implementation,
 and is now basically required by any new functionality in DNS such as
 DNSSEC ([RFC 6891]).
 
-[RFC 6891, section 6.1.2] states that any OPTION-CODE values not understood by a
-responder or requestor MUST be ignored. Unknown OPTION-CODE values must be
-processed as though the OPTION-CODE was not even there.
+[RFC 6891][RFC 6891, section 6.1.2], section 6.1.2, states that any OPTION-CODE values
+not understood by a responder or requestor MUST be ignored. Unknown OPTION-CODE values
+must be processed as though the OPTION-CODE was not even there.
 
-In this test case, we will query  with an unknown EDNS OPTION-CODE and expect
+In this test case, we will query with an unknown EDNS OPTION-CODE and expect
 that the OPTION-CODE is not present in the response for the query.
 
 ## Scope
 
-It is assumed that *Child Zone* has been tested by [Basic04]. This test
-case will set DEBUG level on messages for non-responsive name servers.
+It is assumed that *Child Zone* has been tested and reported by [Basic04]. This
+test case will just ignore non-responsive name servers or name servers not
+giving a correct DNS response for an authoritative name server.
+
+It is assumed that *Child Zone* has been tested and reported by [Nameserver02].
+Running this test case without running [Nameserver02] can give an incomplete
+report status of *Child Zone*.
 
 ## Inputs
 
 "Child Zone" - The domain name to be tested.
 
-## Ordered description of steps to be taken to execute the test case
+## Summary
+
+Message                                | Level     | Arguments           | Description of when message tag is outputted
+:--------------------------------------|:----------|---------------------|---------------------------------------------
+NS11_UNKNOWN_OPTION_CODE               | WARNING   | ns_ip_list          | The DNS response contains an unknown EDNS option-code
```

Comment:
Include the argument and write a true msgid.

Reply:
Fixed